vm/message/src/lib.rs

```diff
@@ -31,3 +33,9 @@ pub trait Message {
     /// Returns the gas limit for the message
     fn gas_limit(&self) -> &BigUint;
 }
+
+#[derive(Serialize, Deserialize)]
```

Comment:
Note that if you derive this will be in map format, when it seems it's serialized as a tuple in Lotus like: `(bls_message_root, secp_message_root)` 

Probably worth implementing here, or in another PR where all of this is tested against Lotus